frontend/public/components/overview/hpa-overview.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { HorizontalPodAutoscalerKind } from '../../module/k8s';
+import { ResourceLink, SidebarSectionHeading } from '../utils';
+import { HorizontalPodAutoscalerModel } from '../../models';
+
+type HPAOverviewProps = {
+  hpas?: HorizontalPodAutoscalerKind[];
+};
+
+export const HPAOverview: React.FC<HPAOverviewProps> = ({ hpas }) => {
+  if (!hpas?.length) {
+    return null;
+  }
+
+  return (
+    <div>
```

Comment:
do we need this `div` here?

Reply:
Nope, was just being lazy - the component was done very quick and I didn't re-evaluate it after the first implementation was done :P